utility/pck_dumper.cpp

```diff
@@ -120,7 +120,7 @@ Error PckDumper::_pck_dump_to_dir(
 			error_string += files.get(i)->get_path() + " (FileAccess error)\n";
 			continue;
 		}
-		String target_name = dir.path_join(files.get(i)->get_path().replace("res://", ""));
+		String target_name = dir.path_join(files.get(i)->get_path().replace("res://", "").replace("user://", ""));
```

Comment:
I have never run into a PCK file that contains a file with a `user://` path, and I don't think the exporter is actually capable of doing it. Have you encountered this in the wild?

Reply:
Yes, I indeed have PCK containing these paths. Why would I try to add it otherwise...

Reply:
![user_paths](https://user-images.githubusercontent.com/63057995/236669618-3e03f502-d2e9-4cf7-b2a7-d1b92352850c.png)

Reply:
I stand corrected! I wonder how the hell they actually got them in the PCK; could you hit me up on the Godot discord to discuss? I'm Nikitalita#3879

Reply:
invite for the Godot discord is here: https://discord.gg/q4GTNxWW